File: WEEK-2/week2d.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
#define pi 3.14
int main()
{
    int choice, r, h;
    float area, volume;
    printf("Enter shape you want\n");
    do
    {
        printf("\nmenu\n 1:Cylider\n 2:Cone\n 3:Sphere\n 4:Exit\n");
        scanf("%d", &choice);
        switch (choice)
        {
        case 1:
            printf("Enter radius:\n");
            scanf("%d", &r);
            printf("Enter height:\n");
            scanf("%d", &h);
            area = (2 * pi * r * h) + (2 * pi * pow(r, 2));
            volume = pi * pow(r, 2) * h;
            printf("Area:%f \t\t Volume:%f", area, volume);
            break;
        case 2:
            printf("Enter radius:\n");
            scanf("%d", &r);
            printf("Enter height:\n");
            scanf("%d", &h);
            area = pi * r * (r + sqrt(pow(h, 2) + pow(r, 2)));
            volume = pi * pow(r, 2) * h / 3.0;
            printf("Area:%f \t\t Volume:%f", area, volume);
            break;
        case 3:
            printf("Enter radius:\n");
            scanf("%d", &r);
            area = 4 * pi * pow(r, 2);
            volume = (4 / 3.0) * (pi * pow(r, 3));
            printf("Area:%f \t\t Volume:%f", area, volume);
            break;
        case 4:
            printf("Exit\n");
            break;
        default:
            printf("Enter a no. ranging from 1 to 4");
        }
    } while (choice != 4);
    return 0;
}

File: WEEK-2-IN-JAVA/week2d.java
import java.util.Scanner;
import java.lang.Math;
public class week2d {
    public static void main(String args[]){
        final double pi = 3.14;
        double r, h;
        int choice;
    double area, volume;
    System.out.println("Enter shape you want\n");
    Scanner sc = new Scanner(System.in);
    do
    {
        System.out.println("menu 1:Cylider 2:Cone 3:Sphere 4:Exit\n");
        choice = sc.nextInt();
        switch (choice)
        {
        case 1:
            System.out.println("Enter radius:\n");
            r = sc.nextDouble();
            System.out.println("Enter height:\n");
            h = sc.nextDouble();
            area = (2 * pi * r * h) + (2 * pi * (Math.pow(r, 2)));
            volume = pi * (Math.pow(r, 2)) * h;
            System.out.println("Area:"+ area +" Volume:"+ volume+"\n");
            break;
        case 2:
            System.out.println("Enter radius:\n");
            r = sc.nextDouble();
            System.out.println("Enter height:\n");
            h = sc.nextDouble();
            area = pi * r * (r + Math.sqrt(Math.pow(h, 2) + Math.pow(r, 2)));
            volume = pi * Math.pow(r, 2) * h / 3.0;
            System.out.println("Area:"+area+" Volume:"+ volume+"\n");
            break;
        case 3:
            System.out.println("Enter radius:\n");
            r = sc.nextDouble();
            area = 4 * pi * Math.pow(r, 2);
            volume = (4 / 3.0) * (pi * (Math.pow(r, 3)));
            System.out.println("Area:"+area+" Volume:"+ volume+"\n");
            break;
        case 4:
            System.out.println("Exit\n");
            sc.close();
            break;
        default:
            System.out.println("Enter a no. ranging from 1 to 4\n");
        }
        
    } while (choice != 4);
    }
}
